cfme/utils/appliance/__init__.py

```diff
@@ -2597,11 +2605,34 @@ def disable_migration_ui(self):
             self. _switch_migration_ui(False)
 
 
+class RegularAppliance(IPAppliance):
```

Comment:
Shouldn't this be named DefaultAppliance now?

Reply:
If so, imports will need to be update

Reply:
@mshriver, do also think this is necessary ?

Reply:
I think its `ok` to name it as `RegularAppliance` pointing `default`

Reply:
We can do it in separate PR if this delays the merge

Reply:
done